### ISSUE TITLE
decode loss bug ?

### ISSUE THREAD

**Issue by danpanwan** [sshaoshuai/MTR#18]
layer_loss = loss_reg_gmm * weight_reg + loss_reg_vel * weight_vel + loss_cls.sum(dim=-1) * weight_cls,   it should be layer_loss = loss_reg_gmm * weight_reg + loss_reg_vel * weight_vel + loss_cls* weight_cls.   'loss_cls.sum(dim=-1)' is wrong here.  Do you agree with me ?

**Comment by sshaoshuai** [comment]
Yes. It is not very formal but it is not an error, since it just adopts different loss weights. 
Actually, I made a mistake here when I refactored the codes previously. But then I just found that it achieved better performance, so I just kept it here. 
One can also rewrite it to the correct form and then tune the loss weights.
